dbmanager/dbmanager.go

```diff
@@ -0,0 +1,93 @@
+package dbmanager
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	_ "github.com/lib/pq" //db driver for postgres
+	"golang.org/x/crypto/bcrypt"
+)
+
+var ErrUserNonexistant = errors.New("User does not exist")
+
+type DBManager struct {
+	dbUser, dbPass, dbName string
+	DB                     *sql.DB
+}
+
+//New creates a new DBManager instance and returns it
+func New(user, pass, name string) (DBManager, error) {
+	d := DBManager{dbUser: user, dbPass: pass, dbName: name}
+	err := d.connectToPSQL()
+	return d, err
```

Comment:
Is there a reason to return db manager if we failed to connect?

Reply:
We could add an error check (previously had one)  and return nil or an empty struct but I don't think it really matters. Is there any reason not to return it as is?